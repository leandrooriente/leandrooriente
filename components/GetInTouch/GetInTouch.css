.link {
  margin: 0 10px 10px 0;
}

.link:last-child {
  margin: 0;
}

@media screen and (max-width: 468px) {
  .links {
    display: flex;
    flex-direction: column;
  }

  .link, .link:last-child {
    margin: 0 auto 10px 0;
  }
}
