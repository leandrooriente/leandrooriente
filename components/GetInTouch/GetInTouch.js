import React from 'react';
import { social } from 'constants';
import Subtitle from 'components/Subtitle';
import Button from 'components/Button';
import css from './GetInTouch.css';

const GetInTouch = () => (
  <div className={css.container}>
    <Subtitle>Get in touch</Subtitle>
    <div className={css.links}>
      {social.map(link => (
        <Button
          className={css.links}
          href={link.url}
          key={link.description}
        >
          {link.name}
        </Button>
      ))}
    </div>
  </div>
);

export default GetInTouch;
