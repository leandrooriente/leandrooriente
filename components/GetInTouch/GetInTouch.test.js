import React from 'react';
import { mount } from 'enzyme';
import GetInTouch from './GetInTouch';

describe('<GetInTouch />', () => {
  it('should render <GetInTouch /> component', () => {
    const wrapper = mount(<GetInTouch />);

    expect(wrapper).toMatchSnapshot();
  });
});
