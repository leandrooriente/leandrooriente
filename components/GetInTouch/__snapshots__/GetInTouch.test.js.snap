// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GetInTouch /> should render <GetInTouch /> component 1`] = `
<GetInTouch>
  <div
    className="container"
    id="get-in-touch"
  >
    <Subtitle>
      <h3
        className="title"
      >
        Get in touch
      </h3>
    </Subtitle>
    <div
      className="links"
    >
      <Button
        className="link"
        href="https://www.linkedin.com/in/leandrooriente/"
        key="Linkedin"
        onClick={null}
      >
        <Link
          href="https://www.linkedin.com/in/leandrooriente/"
        >
          <a
            className="button link"
            href="https://www.linkedin.com/in/leandrooriente/"
            onClick={[Function]}
          >
            linkedin.com/in/leandrooriente
          </a>
        </Link>
      </Button>
      <Button
        className="link"
        href="https://github.com/leandrooriente"
        key="Github"
        onClick={null}
      >
        <Link
          href="https://github.com/leandrooriente"
        >
          <a
            className="button link"
            href="https://github.com/leandrooriente"
            onClick={[Function]}
          >
            github.com/leandrooriente
          </a>
        </Link>
      </Button>
      <Button
        className="link"
        href="mailto:leandroriente@gmail.com"
        key="Email"
        onClick={null}
      >
        <Link
          href="mailto:leandroriente@gmail.com"
        >
          <a
            className="button link"
            href="mailto:leandroriente@gmail.com"
            onClick={[Function]}
          >
            leandroriente@gmail.com
          </a>
        </Link>
      </Button>
      <Button
        className="link"
        href="#"
        key="Resume"
        onClick={null}
      >
        <Link
          href="#"
        >
          <a
            className="button link"
            href="#"
            onClick={[Function]}
          >
            resume
          </a>
        </Link>
      </Button>
    </div>
  </div>
</GetInTouch>
`;
